File: Python-Übungen/primzahlen.py
y = int(input("Bis zu welcher Zahl sollen die Primzahlen ausgegeben werden?\n"))
class num():
    def __init__(self, i):
        self.numb = i
        self.is_prime_num = True
        

if y == 0 or y == 1:
    print("Keine Primzahlen vorhanden")
else:
    print("\n\n\nPrimzahlen: \n\n")
    for i in range(2, y+1, 1):
        x = num(i)
        for j in range(2, int(i/2)+1, 1):
            if x.numb % j == 0:
                x.is_prime_num = False
                break
        if x.is_prime_num == False:
            continue
        else:
            print(f"{x.numb}")
            continue
            
    

File: Semester 1/C-Uebungen/primzahlen.c
#include <stdint.h>
#include <stdio.h>

int main() {
int x = scanf("Bis zu welcher zahl sollen die Primzahlen ausgegeben werden? \n");
int is_prime_num = 1;

if (x == 0 || x == 1) {printf("Keine Primzahlen vorhanden");}
else {

    printf("\n\n\nPrimzahlen\n\n");
    for (int i = 2; x+1; i++) {
        for (int j = 2; (x/2)+1; 1) {
            if (i % j == 0) {
                is_prime_num = 0;
                break;
            }
            else {
                continue;
            }


        }
        if (is_prime_num == 0) {
            continue;
        }
        else {
            printf("%d\n", i);
        }

    }


}






}

